e2e/bootstrap_test.go

```diff
@@ -78,4 +82,85 @@ func testBootstrap() {
 			return nil
 		}).Should(Succeed())
 	})
+
+	It("should record events", func() {
+		cluster, err := getMySQLCluster()
+		Expect(err).ShouldNot(HaveOccurred())
+		stdout, stderr, err := kubectl("get", "-n", "e2e-test", "events", "--field-selector", "involvedObject.name="+cluster.Name, "-o", "json")
+		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
+
+		var events corev1.EventList
+		err = json.Unmarshal(stdout, &events)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		Expect(events.Items).ShouldNot(HaveLen(0))
```

Comment:
Should check `Initialization Succeeded` Event and another Event are included (because `Initialization Succeeded` Event is not checked by small tests).